pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -260,13 +260,10 @@ func (r *Builder) PodEnvironment(vmRef ref.Ref, sourceSecret *core.Secret) (env
 }
 
 func (r *Builder) mapMacStaticIps(vm *model.VM) (ipMap string, err error) {
-	// on windows machines we check if the interface origin is manual
-	// on linux we collect all networks.
-	isWindowsFlag := isWindows(vm)
```

Comment:
why don't we need this? 
tbh I don't remember why we needed it in the first place, @yaacov do you remember by any chance?

Reply:
on windows we have the `origin` and we don't need to invent it's value buy guessing

Reply:
now that we have code that "invent" the "origin" for none windows vms, we can omit this check, because we have the origin for all machines (in case we find a nic that match the mac of a network)